README.md

```diff
@@ -21,12 +21,12 @@ BASE_URL = http://localhost:3000
 
 * create an account at [cloudinary.com](https://cloudinary.com) and create a folder called campgrounds. Copy/paste your cloudinary credentials to the .env file
 
-```.env
-DB_URL = mongodb://<dbuser>:<dbpassword>@ds111476.mlab.com:11476/<db_name>
-BASE_URL = http://localhost:3000
-cloud_name = <cloud_name_here>
-api_key = <api_key_here>
-api_secret = <api_secret>
+```bash
+DB_URL='mongodb://<dbuser>:<dbpassword>@ds111476.mlab.com:11476/<db_name>'
+BASE_URL='http://localhost:3000'
+cloud_name='<cloud_name_here>'
+api_key='<api_key_here>'
+api_secret='<api_secret>'
 ```
```

Comment:
I faced issues when i put quotes for string literals in the env. Are you sure this works?

Reply:
Yep it does. I use it all the time. String either the way